packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -326,6 +326,7 @@ const AuthAndAccountSetupRoutes = ({
         path="/signup_verified/*"
         {...{ isSignedIn, serviceName }}
       />
+      <SigninContainer path="/force_auth/*" {...{ integration, serviceName }} />
```

Comment:
Just a small nit, we've been organizing these by component name (`SigninContainer`) and then by `path`. I guess that means this should go above `path="confirm_signup_code"`.